hermes/README.md

```diff
@@ -0,0 +1,16 @@
+<!-- cspell: words indexmap -->
+
+# Hermes core
+
+An implementation of the Hermes core engine in Rust
+
+## Build notes
+
+Unfortunately during the build process you could face with the known problem
```

Comment:
Can we update this comment by removing emotional aspects?

for example:

During the build process, you may encounter specific known issues: [tower/issues/466](https://github.com/tower-rs/tower/issues/466) and [indexmap/issues/151](https://github.com/indexmap-rs/indexmap/issues/151). These issues can impede the build's success. We recommend explicitly setting the environment variable `CARGO_FEATURE_STD=1 cargo b` as a temporary solution. This workaround has effectively bypassed the mentioned problems until a permanent fix is implemented.

Reply:
fixed